rfc8446.xml

```diff
@@ -1148,7 +1153,8 @@ negotiate the protocol version. All TLS&nbsp;1.3 ServerHello messages MUST
 contain the "supported_versions" extension.  Current ServerHello messages
 additionally contain either the "pre_shared_key" extension or the "key_share"
 extension, or both, when using a PSK with (EC)DHE key establishment. Other
-extensions are sent separately in the EncryptedExtensions message.</t>
+extensions  (see  <xref target="extensions"/>)
```

Comment:
I don't *think* the extra space here makes it into the rendered text...

Reply:
I certainly hope not.